ui/admin/app/components/form/field/list-wrapper/text-input/index.hbs

```diff
@@ -0,0 +1,65 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::Form::Fieldset
+  @layout={{@layout}}
+  @isRequired={{@isRequired}}
+  @isOptional={{@isOptional}}
+  name={{@name}}
+  as |F|
+>
+
+  {{yield F}}
+
+  <F.Control>
+
+    {{#each @options as |option index|}}
+
+      <Hds::SegmentedGroup class='list-wrapper-input' as |S|>
```

Comment:
Should we be using segmented group? The looks are now inconsistent, plus the add button and delete button don't line up

Reply:
I think we can fix the style for the existing component if we can use segmented group. I reached out to Joshua to see if we can use this component, technically it doesn't have the label and satisfies the requirements.